ARCHITECHTURE.md

```diff
@@ -853,7 +827,7 @@ export class YourMemoryService extends BaseMemoryService {
 
 Create `packages/adk/src/artifacts/your-artifact-service.ts`:
 
-```typescript
+````typescript
```

Comment:
There is an extra tick here that is causing the next section to be rendered as code.